crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -57,6 +64,27 @@ def knife_files
           new_file = filename_replace(file_path, "nova_dashboard", "horizon")
           filecontent_replace(new_file, "nova_dashboard", "horizon")
           file_path = new_file
+        when /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          # override with migrated attributes
+          attributes = SchemaMigration.migrated_attributes("network", json)
+          if attributes.nil?
+            @status[:status] = :internal_server_error
+            @status[:errors].push "Could not upgrade the network proposal with #{file.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)